pkg/apis/pipeline/v1beta1/matrix_types.go

```diff
@@ -72,23 +72,23 @@ func (m *Matrix) FanOut() []Params {
 	for _, parameter := range m.Params {
 		combinations = combinations.fanOutMatrixParams(parameter)
 	}
-	combinations = combinations.overwriteCombinations(includeCombinations)
+	combinations.overwriteCombinations(includeCombinations)
 	combinations = combinations.addNewCombinations(includeCombinations)
 	return combinations.toParams()
 }
 
-// overwriteCombinations replaces any missing include params in the initial
-// matrix params combinations by overwriting the initial combinations with the
-// include combinations
-func (cs Combinations) overwriteCombinations(ics Combinations) Combinations {
+func (cs Combinations) overwriteCombinations(ics Combinations) {
```

Comment:
shall we keep the doc string here?

Reply:
Thanks @JeromeJu! Added back the docstring.